bridges/QPlayBridge.php

```diff
@@ -0,0 +1,133 @@
+<?php
+class QPlayBridge extends BridgeAbstract {
+	const NAME = 'Q Play';
+	const URI = 'https://www.qplay.pt';
+	const DESCRIPTION = 'Entretenimento e humor em Português';
+	const MAINTAINER = 'somini';
+	const PARAMETERS = array(
+		'Program' => array(
+			'program' => array(
+				'name' => 'Program Name',
+				'type' => 'text',
+				'required' => true,
+			),
+		),
+		'Catalog' => array(
+			'all_pages' => array(
+				'name' => 'All Pages',
+				'type' => 'checkbox',
+				'defaultValue' => false,
+			),
+		),
+	);
+
+	public function getIcon() {
+		$html = getSimpleHTMLDOMCached(self::URI)
+			or returnServerError('Could not load content');
+
+		return $html->find('head link[rel="apple-touch-icon"]', 0)->getAttribute('href');
+	}
+
+	public function getURI() {
+		switch ($this->queriedContext) {
+			case 'Program':
+				return self::URI . '/programs/' . $this->getInput('program');
+			case 'Catalog':
+				return self::URI . '/catalog';
+		}
+		return parent::getURI();
+	}
+
+	public function getName() {
+		switch ($this->queriedContext) {
+			case 'Program':
+				$html = getSimpleHTMLDOMCached($this->getURI())
+					or returnServerError('Could not load content');
+
+				return $html->find('h1.program--title', 0)->innertext;
+			case 'Catalog':
+				return self::NAME . ' | Programas';
+		}
+
+		return parent::getName();
+	}
+
+	/* This uses the uscreen platform, other sites can adapt this. https://www.uscreen.tv/ */
+	public function collectData() {
+		switch ($this->queriedContext) {
+		case 'Program':
+			$program = $this->getInput('program');
+			$html = getSimpleHTMLDOMCached($this->getURI())
+				or returnServerError('Could not load content');
+
+			foreach($html->find('.cce--thumbnails-video-chapter') as $element) {
+				$cid = $element->getAttribute('data-id');
+				$item['title'] = $element->find('.cce--chapter-title', 0)->innertext;
+				$item['content'] = $element->find('.cce--thumbnails-image-block', 0) . $element->find('.cce--chapter-body', 0)->innertext;
```

Comment:
So this line is too long. Here is my suggestion for a fix.

```suggestion
				$item['content'] = $element->find('.cce--thumbnails-image-block', 0)
				. $element->find('.cce--chapter-body', 0)->innertext;
```

Essentially use the concatenation for a line break.